src/pyspiffe/workloadapi/exceptions.py

```diff
@@ -75,3 +75,17 @@ def __init__(self, additional_information: str = 'none') -> None:
             additional_information: Additional information about the error.
         """
         super().__init__(self._MESSAGE.format(additional_information))
+
+
+class X509SourceError(WorkloadApiError):
+    """Error raised when there is an error in the X.509 Source."""
+
+    _MESSAGE = 'X.509 Source error: {}.'
```

Comment:
nit: final dot is set automatically so we removed it from all the custom exceptions.